manager/controllers/baselines_test.go

```diff
@@ -29,9 +29,9 @@ func TestBaselinesDefault(t *testing.T) {
 	output := testBaselines(t, "/")
 
 	assert.Equal(t, 2, len(output.Data))
-	assert.Equal(t, 2, output.Data[0].ID)
-	assert.Equal(t, "baseline", output.Data[0].Type)
-	assert.Equal(t, "baseline_1-2", output.Data[0].Attributes.Name)
+	assert.Equal(t, 1, output.Data[1].ID)
```

Comment:
That's it. This test depends on the previous one because checked data are impacted. We should avoid this.